flow/layers/color_filter_layer_unittests.cc

```diff
@@ -82,10 +87,10 @@ TEST_F(ColorFilterLayerTest, SimpleFilter) {
   const SkRect child_bounds = SkRect::MakeLTRB(5.0f, 6.0f, 20.5f, 21.5f);
   const SkPath child_path = SkPath().addRect(child_bounds);
   const SkPaint child_paint = SkPaint(SkColors::kYellow);
-  auto layer_filter =
-      SkColorMatrixFilter::MakeLightingFilter(SK_ColorGREEN, SK_ColorYELLOW);
+  auto dl_color_filter = DlColorFilter::From(
```

Comment:
I don't think this test was testing the behavior of a Lighting filter, it's just a basic test of using the color filter layer with a basic filter. It would be good to switch it to some variation that we support in DlColorFilter.

Reply:
Using DlColorFilter